tests/integration/run_tests.py

```diff
@@ -35,24 +36,36 @@
 ]
 
 
-def get_target(silk: bool, method: str):
+def get_target(target_type: str, method: str):
     """ determine target
     """
     if "engine_" in method:
         return "localhost:8550"
-    if silk:
+    if target_type == SILK:
         return "localhost:51515"
 
+    if target_type == INFURA:
+        return "https://goerli.infura.io/v3/d1c30a8114fd443aad1ed5d4b403e33d"
```

Comment:
This URL must be passed as input argument and *NOT* hard-coded because:
- it is specific for Goerli network whilst this run script is not
- it contains personal account information